src/components/ebay-breadcrumb/template.marko

```diff
@@ -0,0 +1,12 @@
+<if(data.items)>
+    <nav aria-labelledby="breadcrumbs_heading" class="breadcrumb" role="navigation" ${data.htmlAttributes} w-bind w-onclick=(data.preventDefault && 'handleClick') ${data.preventDefault}>
+        <h2 id="breadcrumbs_heading" class="clipped">${data.ariaLabel}</h2>
+        <ul>
+            <for(item in data.items)>
+                <li if(item)>
+                    <${item.tag} ${item.htmlAttributes} w-body=item.renderBody></${item.tag}>
```

Comment:
(minor) You can use `/>` instead of the closing tag (since it has no inner content).